hack/ci/e2e-olm.sh

```diff
@@ -82,30 +177,212 @@ function check_spo_is_running() {
     # (jhrozek): I didn't find a useful condition or status to wait for..
     # ..if only there was a way to check if ANY installedCSV is set..
     sleep 30
-    CSV=$(kubectl -nsecurity-profiles-operator get sub security-profiles-operator-sub -ojsonpath='{.status.installedCSV}')
+    CSV=$(kubectl -n$ns get sub security-profiles-operator-sub -ojsonpath='{.status.installedCSV}')
     # wait for the CSV to be actually installed
-    kubectl -nsecurity-profiles-operator wait --for=jsonpath='{.status.phase}'=Succeeded csv $CSV
+    kubectl -n$ns wait --for=jsonpath='{.status.phase}'=Succeeded csv $CSV
 
     # wait for the operator to be ready
-    kubectl -nsecurity-profiles-operator wait --for=condition=ready pod -lname=security-profiles-operator
-    kubectl -nsecurity-profiles-operator wait --for=condition=ready pod -lname=security-profiles-operator-webhook
+    kubectl -n$ns wait --for=condition=ready pod -lname=security-profiles-operator || return 1
+
+    # wait for webhook deploy to be created, kubectl wait for non-existent resource seems to exit with error
+    # which is causing random test failure
+    # see https://github.com/kubernetes/kubernetes/issues/83242
+    for i in $(seq 1 10); do
+        found=$(kubectl -n$ns wait --for=condition=ready pod -lname=security-profiles-operator-webhook 2>/dev/null)
+        if [[ $found ]]; then
+            break
+        fi
+        sleep 5
+    done
+    kubectl -n$ns wait --for=condition=ready pod -lname=security-profiles-operator-webhook || return 1
 
-    # wait for spod pod to be created, kubectl wait for un-existed resource seems to exit with error
+    # wait for spod pod to be created, kubectl wait for non-existent resource seems to exit with error
     # which is causing random test failure
     # see https://github.com/kubernetes/kubernetes/issues/83242
     for i in $(seq 1 10); do
-        found=$(kubectl get -nsecurity-profiles-operator pods -lname=spod 2>/dev/null)
+        found=$(kubectl get -n$ns pods -lname=spod 2>/dev/null)
         if [[ $found ]]; then
             break
         fi
         sleep 5
     done
-    kubectl -nsecurity-profiles-operator wait --for=condition=ready pod -lname=spod
+    kubectl -n$ns wait --for=condition=ready pod -lname=spod || return 1
+
+    return 0
+}
+
+function assert_spo_csv_installed_in_ns() {
+    ns=$1
+
+    [[ $(kubectl get csv -loperators.coreos.com/security-profiles-operator.$ns= -n$ns -oname) ]] || return 1
```

Comment:
this probably looks hacky, but I found it easier to list any CSV in the NS that installs SPO rather than parse out the version of the CSV directly..
the `-loperators.coreos.com/security-profiles-operator.$ns=` is an OLM-ism, the label is `-loperators.coreos.com/$NAME_OF_THE_OPERATOR.$NAMESPACE_WHERE_THE_OPERATOR_IS_INSTALLED=`